src/bunfig.zig

```diff
@@ -252,6 +253,26 @@ pub const Bunfig = struct {
                         this.ctx.test_options.coverage.enabled = expr.data.e_boolean.value;
                     }
 
+                    if (test_.get("coverageReporters")) |expr| {
+                        try this.expect(expr, .e_array);
+                        const items = expr.data.e_array.items.slice();
+                        const reporters = try this.allocator.alloc(TestCommand.Reporter, items.len);
+                        for (items, 0..) |item, i| {
+                            try this.expectString(item);
+                            if (TestCommand.ReporterMap.get(item.asString(bun.default_allocator) orelse "")) |reporter| {
+                                reporters[i] = reporter;
+                            } else {
+                                try this.addError(item.loc, "Invalid reporter.");
```

Comment:
can this say `Invalid coverage reporter "{s}"`?

Reply:
Did so in https://github.com/oven-sh/bun/pull/11883/commits/a426f3e45eff522f65aa4c6596d15726d37398dc